LabExT/Movement/Calibration.py

```diff
@@ -71,6 +71,8 @@ def __init__(self, mover, stage, orientation, device_port) -> None:
         self.mover = mover
         self.stage: Type[Stage] = stage
 
+        self.stage_polygon: Type[StagePolygon] = DCProbe(orientation)
```

Comment:
For now it is hard coded, different polygons can be selected later in the `StageSetup` wizard.